### DISCUSSION TITLE
Plotman not tracking phase, plot id and tmp values of manually ran plot jobs

### DISCUSSION THREAD

**Discussion by flmst** [ericaltendorf/plotman#491] [General]
Hi,

Is it possible for plotman to track _phase_, _plot id_ and _tmp_ values of manually created plot jobs? At the moment plots are recognized but mentioned values are missing:

 ```
Prefixes:  tmp=/mnt/ssd1  dst=/mnt/hdd (remote)
  #    plot id    k              tmp          dst    wall   phase    tmp      pid   stat    mem    user    sys     io
  0   --------   32       /mnt/ssd3/   bigdata5-b    0:29     ?:?      0   144465    SLP   4.8G    0:55   0:03     0s
  1   7176af5c   32                .   bigdata5-b    0:51     1:3   117G   143119    SLP   5.3G    1:55   0:06     1s
  2   54f84a15   32                .   bigdata5-c    0:53     1:3   120G   142655    SLP   5.3G    1:59   0:06     2s
 ```

I use two slower SSDs that I don't want to assign the same CPU per plot as for the main SSD (so slow SSDs are not hogging CPU for extended amount of time while doing ph1).

Is it a logfile in a certain format that would make plotman display that info? Or should I change the command used to manually run a plot? Currently I use this:

`screen -d -m -S chia3-1 bash -c 'cd /home/toom/chia-blockchain && . ./activate && sleep 0h && chia plots create -k 32 -b 4000 -e -r 5 -u 128 -t /mnt/ssd3/ -d /mnt/hdd/bigdata5-b | tee /home/toom/chia-blockchain/logs/chia3-1-'date +%d%b%Y-%H%m'.log' `

EDIT:
I ran the job with command mimicking plotman (taken from ps -ef) but it didn't make a difference.

` /home/toom/chia-blockchain/venv/bin/python /home/toom/chia-blockchain/venv/bin/chia plots create -k 32 -r 5 -u 128 -b 4000 -t /mnt/ssd3/ -d /mnt/hdd/bigdata5-c`

**Comment by altendky** [comment]
Ok, so the issue there is actually one I ran into the other day myself.  We look for log files open by the plot process itself.  When you `>` the process to the left of it has the file open.  When you `| tee` then `tee` has the file open instead.  So, just `>` to the file and it should work.  Note that you can monitor the file from the shell if you want using `tail -f the_file`.

**Reply by flmst** [reply]
Thanks! As suggested, swapping`  | tee` for `>` fixed the issue.
